app/bundles/PageBundle/Translations/en_US/messages.ini

```
@@ -164,6 +164,7 @@ mautic.widget.unique.vs.returning.leads="Unique vs returning visitors"
 mautic.widget.dwell.times="Dwell times"
 mautic.widget.popular.pages="Popular landing pages"
 mautic.widget.created.pages="Created Landing pages"
+mautic.widget.best.tracking.pages="Best tracking pages"
```

```suggestion
mautic.widget.best.tracking.pages="Popular external pages"
```

Based on the feedback from the UX folks, I think this is the best term to use. We need to be clear this relates to external pages, not landing pages, so folks understand the difference. Thanks @code5rick and @andersonjeccel for the input.

---

We used to use the word "tracking" in communication (docs, forums, etc.). This decision makes me wonder.